addon/components/quarterly-closure/disabled-form-info.hbs

```diff
@@ -1,4 +1,2 @@
-<div uk-alert class="uk-alert-warning">
-  <FaIcon @icon="circle-info" @fixedWidth={{true}} class="margin-right-xs" />
-  {{t "ember-gwr.quarterlyClosure.components.disabledFormInfo"}}
-</div>
+<FaIcon @icon="circle-info" @fixedWidth={{true}} class="margin-right-xs" />
```

Comment:
Is it intentional, that the text and icon are black for linked-models?

Reply:
I've added a screenshot, yes it is intentional. I think it is not needed to have a flashy text there.

Reply:
Yes, I took a look at it and I agree :+1: